app/test_main.py

```diff
@@ -1,3 +1,28 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestHumanAgeConversion:
+    @pytest.mark.parametrize("cat_age, dog_age, expected", [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+    ])
+    def test_get_human_age(
+            self,
+            cat_age: int,
+            dog_age: int,
+            expected: list[int]
+    ) -> None:
+        assert get_human_age(cat_age, dog_age) == expected
+
+    def test_non_integer_age(self) -> None:
+        with pytest.raises(TypeError):
+            get_human_age("cat", 10)
+        with pytest.raises(TypeError):
+            get_human_age(5, "dog")
```

Comment:
you should parametrize this + add different data types that may cause an error. You have str + int and int + str which is basically same